src/ignoreButRetain.js

```diff
@@ -308,6 +345,11 @@ define([
                     return attrs;
                 });
             },
+            getSetValues: function(mug) {
+                return _.map(mug.p.setValues, function(setValue) {
+                    return setValue;
+                });
```

Comment:
Could this be changed to `return mug.p.setValues;`? It doesn't look like the `map` is doing anything.